solo/solo.py

```diff
@@ -178,248 +127,121 @@ def main():
                 'ignore_batch']:
         if par in params:
             vae_params[par] = params[par]
-    vae_params['n_batch'] = 0 if params.get(
-        'ignore_batch', False) else scvi_data.n_batches
 
     # training parameters
+    batch_key = params.get('batch_key', None)
     batch_size = params.get('batch_size', 128)
     valid_pct = params.get('valid_pct', 0.1)
     learning_rate = params.get('learning_rate', 1e-3)
-    stopping_params = {'patience': params.get('patience', 10), 'threshold': 0}
+    stopping_params = {'patience': params.get('patience', 100), 'min_delta': 0}
```

Comment:
Does this mean 100 epochs? That seems too large

Reply:
@davek44 I upped it because I have the patience to adjust the learning rate set to 20, and I shrink the learning rate from `1e-2` to `1e-7`. I can adjust both though to be less. A patience of 5 for adjusting the learning rate and a global patience of 30?

Reply:
How about 10 for the learning rate and 20 to stop once and for all? Also, you typically don't decrease the learning rate quite that much. I would guess 100x decrease is about right.